scripts/bracket_orders.js

```diff
@@ -45,29 +48,82 @@ const argv = require("yargs")
   )
   .version(false).argv
 
+// returns undefined if the price was not available
+const getDexagPrice = async function (tokenBought, tokenSold) {
+  // dex.ag considers WETH to be the same as ETH and fails when using WETH as token
+  tokenBought = tokenBought == "WETH" ? "ETH" : tokenBought
+  tokenSold = tokenSold == "WETH" ? "ETH" : tokenSold
+  // see https://docs.dex.ag/ for API documentation
+  const url = "https://api-v2.dex.ag/price?from=" + tokenSold + "&to=" + tokenBought + "&fromAmount=1&dex=ag"
+  let price
+  try {
+    const requestResult = await axios.get(url)
+    price = requestResult.data.price
+  } catch (error) {
+    console.log("Warning: unable to retrieve price information on dex.ag. The server returns:")
+    console.log(">", error.response.data.error)
+  }
+  return price
+}
+
+const thresholdPercent = 2
```

Comment:
Maybe a little bit more descriptive name would be acceptedPriceDeviationInPercentage